src/core/SolarEclipseComputer.hpp

```diff
@@ -0,0 +1,170 @@
+/*
+ * Stellarium
+ * 
+ * Copyright (C) 2015 Alexander Wolf
+ * Copyright (C) 2016 Nick Fedoseev (visualization of ephemeris)
```

Comment:
Maybe reduce Nick? Or is some of his content from AstroCalc in this file?

Reply:
+1 - Yes, the new file doesn't have Nick's code

Reply:
Hmm... my name should be removed from new files too

Reply:
OK, removed Nick and Alex.

Reply:
Sorry to be nitpicky, but now it looks strange. Worachate did most of this and should be first, I refactored a little bit (more possible).

Reply:
Is this OK? (Also removed "(Eclipses)" since the whole file is for eclipses now.)
```
 * Copyright (C) 2022 Worachate Boonplod
 * Copyright (C) 2022 Georg Zotti
 * Copyright (C) 2024 Ruslan Kabatsayev
```